xml/decoders/uhlenbrock/vStartHighMid-accelDecel.xml

```diff
@@ -40,6 +46,7 @@
     <label xml:lang="fr">V démarr.</label>
     <label xml:lang="de">Anfahrspannung</label>
     <tooltip>Speed setting between 1 and 63</tooltip>
+	<tooltip xml:lang="de">Geschwindigkeitseinstellung zwischen 1 und 63</tooltip>	
     <tooltip xml:lang="it">Valori tra 1 e 63</tooltip>
```

Comment:
It seems that you have used TABs instead of spaces for the changes in this file. Or that the original file uses TABs and you are using spaces.If you look at this file on the PR on GitHub, you will see that the lines you have added in this file has a different indentation. That's usually because GitHub uses a different TAB size than you do.